db/schema.rb

```diff
@@ -272,6 +298,9 @@
     t.integer "submission_max_team_size"
     t.integer "submission_grace_period", default: 15
     t.boolean "legacy_seminar", default: false
+    t.integer "annotations_status"
+    t.integer "emergency_link_status", default: 0
+    t.text "emergency_link"
```

Comment:
"emergency" even made it to the database ;) remove emergency branding (see comment above).

Reply:
But in this case the branding "emergency" actually makes sense. What would be your suggestion for renaming it?

Reply:
Yeah, you're right, in this case it does make sense if it's just referring to the one link leading to further resources 👍